gui/app.py

```diff
@@ -48,29 +50,35 @@ def setup_buttons(self):
         self.create_exit_button()
 
     def create_files_button(self):
-        self.load_button_files = customtkinter.CTkButton(self.sidebar, text='Load image from files', font=self.font, command=self.callback_files)
+        self.load_button_files = customtkinter.CTkButton(self.sidebar, text='Load image from files', font=self.font,
+                                                         command=self.callback_files)
         self.load_button_files.grid(row=0, padx=20, pady=20, sticky='ew')
 
     def create_web_button(self):
-        self.load_button_web = customtkinter.CTkButton(self.sidebar, text='Load image from web', font=self.font, command=self.callback_web)
+        self.load_button_web = customtkinter.CTkButton(self.sidebar, text='Load image from web', font=self.font,
+                                                       command=self.callback_web)
         self.load_button_web.grid(row=1, padx=20, pady=10, sticky='ew')
 
     def create_separator(self):
         ttk.Separator(self.sidebar, orient='vertical').grid(row=2, sticky='ew', padx=10, pady=10)
 
     def create_apply_button(self):
-        self.apply_button = customtkinter.CTkButton(self.sidebar, text='CHECK IF CAGE', font=self.font, command=self.callback_apply, state='disabled', fg_color='SpringGreen4')
+        self.apply_button = customtkinter.CTkButton(self.sidebar, text='CHECK IF CAGE', font=self.font,
+                                                    command=self.callback_apply, state='disabled',
+                                                    fg_color='SpringGreen4')
         self.apply_button.grid(row=3, padx=20, pady=10, sticky='ew')
 
     def create_exit_button(self):
-        self.exit_button = customtkinter.CTkButton(self.sidebar, text='Exit', font=self.font, fg_color='purple', hover_color='dark violet', command=self.leave)
+        self.exit_button = customtkinter.CTkButton(self.sidebar, text='Exit', font=self.font, fg_color='purple',
+                                                   hover_color='dark violet', command=self.leave)
         self.exit_button.grid(row=4, padx=20, pady=20, sticky='sew')
 
     def callback_files(self):
-        filename = filedialog.askopenfilename(initialdir=os.environ['HOME'], title='Select image')
+        filename = filedialog.askopenfilename(initialdir=os.path.expanduser('~'), title='Select image')
+        cv2.imwrite('./Faces/detectedImage.jpg', detect_image(cv2.imread(filename), filename))
```

Comment:
not very clear why `detect_image` function is used in such way, returning object is simply product of `cv2.imread`, but function mainly cuts faces of image and saves to files. maybe it should be named differently and be used as procedure